WordPress/Classes/ViewRelated/Post/WPPostViewController.m

```diff
@@ -75,7 +75,8 @@ @interface WPPostViewController () <
 UITextViewDelegate,
 UIViewControllerRestoration,
 EditImageDetailsViewControllerDelegate,
-MediaProgressCoordinatorDelegate
+MediaProgressCoordinatorDelegate,
```

Comment:
This file is for the Hybrid editor, you'll want to make this change in Aztec instead.

Reply:
@catehstn - I'm doing a separate branch for the Aztec toolbar.